mllib/src/test/scala/org/apache/spark/ml/tuning/TrainValidationSplitSuite.scala

```diff
@@ -160,11 +160,13 @@ class TrainValidationSplitSuite
       .setTrainRatio(0.5)
       .setEstimatorParamMaps(paramMaps)
       .setSeed(42L)
+      .setParallelism(2)
```

Comment:
Could you update the test for the Model too please?

Reply:
No. The model do not own `parallel` parameter. This was discussed before.

Reply:
Ah, you're right, thanks